Evaluation.css

```diff
@@ -0,0 +1,105 @@
+body{
+	font-family: "Arial", sans-serif;
+	position: relative;
+  margin: 0;
+  padding-bottom: 6rem;
+  min-height: 100%;
+
+}
+#pageHeader{
+	font-weight: 400;
+	font-size: 24px;
+	list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    width: 100%;
+    box-shadow: 0px 2px 2px #888888;
```

Comment:
I'd argue that `0px`, `0%`, `0vmax`, `0rem` etc are all equivalent and equal to `0`. Would you agree with this statement?